WTWSMS/common/cultures/00_cultures.txt

```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```

Comment:
There is no confusion. Cultures can have their own modifiers that apply to the culture. I didn't want to distribute modifiers to other cultures. The Spartan culture gets a significant Martial boost because that is historical.

I do not appreciate you belittling me by telling me that I haven't read the Wiki or looked into the code.

Given that you issued a thumbs down to my post, you clearly didn't read the code, because I ONLY gave that modifer to the Spartan culture, not the Byzantine Group.

![untitled](https://user-images.githubusercontent.com/17663470/28747554-e5868424-746f-11e7-8f44-f03b5e192160.png)